package.json

```diff
@@ -10,8 +10,8 @@
     "start": "sanity start",
     "build": "sanity build",
     "prettier": "prettier --write '**/*.js'",
-    "deploy-graphql": "sanity graphql deploy --playground",
```

Comment:
I realized trying to troubleshoot how some prod data got overwritten that our command intended to push to push data to the staging graphql was actually going to production. I don't think we noticed before because there's never been much of a difference between the two datasets, but the below changes should fix